airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -93,6 +93,8 @@ def handle_call_rate_limit(self, response, params):
             max_pause_interval = self.pause_interval_minimum
 
             for record in response.json():
+                if "headers" not in record:
```

Comment:
Why would the header not be present? Could you add a comment to help the reader understand?

Reply:
added comment